kombu/simple.py

```diff
@@ -67,7 +68,7 @@ def get(self, block=True, timeout=None):
                 remaining = timeout - elapsed
 
     def get_nowait(self):
-        m = self.queue.get(no_ack=self.no_ack)
+        m = self.queue.get(no_ack=self.no_ack, accept=self.accept)
```

Comment:
Could we avoid this by simply using the `self.consumer.accept` attribute? Then we wouldn't need the new kwarg for this class.

Reply:
Yes I think it is reasonable.